Public/Src/Cache/ContentStore/Vfs/VirtualizedContentSession.cs

```diff
@@ -0,0 +1,152 @@
+        {
+            return InnerSession.PinAsync(operationContext, contentHashes, operationContext.Token, urgencyHint);
+        }
+
+        /// <inheritdoc />
+        protected override Task<PlaceFileResult> PlaceFileCoreAsync(OperationContext operationContext, ContentHash contentHash, AbsolutePath path, FileAccessMode accessMode, FileReplacementMode replacementMode, FileRealizationMode realizationMode, UrgencyHint urgencyHint, Counter retryCounter)
+        {
+            return InnerSession.PlaceFileAsync(operationContext, contentHash, path, accessMode, replacementMode, realizationMode, operationContext.Token, urgencyHint);
+        }
+
+        /// <inheritdoc />
+        protected override Task<IEnumerable<Task<Indexed<PlaceFileResult>>>> PlaceFileCoreAsync(OperationContext operationContext, IReadOnlyList<ContentHashWithPath> hashesWithPaths, FileAccessMode accessMode, FileReplacementMode replacementMode, FileRealizationMode realizationMode, UrgencyHint urgencyHint, Counter retryCounter)
+        {
+            return InnerSession.PlaceFileAsync(operationContext, hashesWithPaths, accessMode, replacementMode, realizationMode, operationContext.Token, urgencyHint);
+        }
+
+        /// <inheritdoc />
+        protected override Task<PutResult> PutFileCoreAsync(OperationContext operationContext, ContentHash contentHash, AbsolutePath path, FileRealizationMode realizationMode, UrgencyHint urgencyHint, Counter retryCounter)
+        {
+            return InnerSession.PutFileAsync(operationContext, contentHash, path, realizationMode, operationContext.Token, urgencyHint);
+        }
+
+        /// <inheritdoc />
+        protected override Task<PutResult> PutFileCoreAsync(OperationContext operationContext, HashType hashType, AbsolutePath path, FileRealizationMode realizationMode, UrgencyHint urgencyHint, Counter retryCounter)
+        {
+            return InnerSession.PutFileAsync(operationContext, hashType, path, realizationMode, operationContext.Token, urgencyHint);
+        }
+
+        /// <inheritdoc />
+        protected override Task<PutResult> PutStreamCoreAsync(OperationContext operationContext, ContentHash contentHash, Stream stream, UrgencyHint urgencyHint, Counter retryCounter)
+        {
+            return InnerSession.PutStreamAsync(operationContext, contentHash, stream, operationContext.Token, urgencyHint);
+        }
+
+        /// <inheritdoc />
+        protected override Task<PutResult> PutStreamCoreAsync(OperationContext operationContext, HashType hashType, Stream stream, UrgencyHint urgencyHint, Counter retryCounter)
+        {
+            return InnerSession.PutStreamAsync(operationContext, hashType, stream, operationContext.Token, urgencyHint);
+        }
+
+        //protected override async Task<PlaceFileResult> PlaceFileCoreAsync(
+        //    OperationContext operationContext,

```

Comment:
Remove?

Reply:
Still debating that with myself. Will discuss with team.

---
In reply to: [280870341](https://github.com/Microsoft/BuildXL/pull/249#discussion_r280870341) [](ancestors = 280870341)